README.md

```diff
@@ -50,27 +73,38 @@ Some key features of Diffuse:
 [rcs]: https://www.gnu.org/software/rcs/
 [svn]: https://subversion.apache.org
 
-## Documentation
+## 📚 Documentation
+
+Comprehensive documentation is available for users, translators, and developers in the [documentation](docs/) directory, covering:
+
+- User guide and tutorials
+- Developer API reference
+- Translation guidelines
+- Build and installation instructions
```

Comment:
If you change the first sentence (it doesn't need it), don't put the list. If you want the list, remove the first sentence, and put links directly to the various parts of the list.